packages/build/src/lib/config/create-config.ts

```diff
@@ -57,9 +70,25 @@ export function createConfig(
               options: {
                 api: 'modern-compiler',
                 implementation: require.resolve('sass-embedded'),
+                ...sassPathOptions,
+                ...(options.stylePreprocessorOptions?.sass ?? {}),
+              },
+            },
+          ],
+          type: 'css/auto',
+        },
+        {
+          test: /\.less$/,
+          use: [
+            {
+              loader: 'less-loader',
+              options: {
+                javascriptEnabled: true,
+                ...lessPathOptions,
               },
             },
           ],
+          // set to 'css/auto' if you want to support '*.module.less' as CSS Modules, otherwise set type to 'css'
```

Comment:
A link to that shows how the context information etc would help. Also, is it a Todo or should we add JSDocs for this behaviour?

Reply:
It is actually probably a Supermaven artifact that I didn't clean up. Removing